Mention usage of alarm_stream_max for TTS notifications (#381)
home-assistant/companion.home-assistant: docs/notifications/basic.md

        data:
          channel: alarm_stream
```

> If you find that your alarm stream volume might be too low you can use `channel: alarm_stream_max` which will temporarily set the alarm stream volume to the max level, speak the notification and then revert back to the original volume level.

Reviewer: ```suggestion
If you find that your alarm stream volume is too low you can use `channel: alarm_stream_max` which will temporarily set the alarm stream volume to the max level, play the notification and then revert back to the original volume level.
```
I don't know the notification but I think this might read more clearly. I might be totally wrong
